src/pages/workspace/accounting/xero/advanced/XeroAdvancedPage.tsx

```diff
@@ -22,14 +22,20 @@ function XeroAdvancedPage({policy}: WithPolicyConnectionsProps) {
     const xeroConfig = policy?.connections?.xero?.config;
     const {autoSync, pendingFields, sync} = xeroConfig ?? {};
     const {bankAccounts} = policy?.connections?.xero?.data ?? {};
-    const {invoiceCollectionsAccountID} = sync ?? {};
+    const {invoiceCollectionsAccountID, reimbursementAccountID} = sync ?? {};
 
     const selectedBankAccountName = useMemo(() => {
         const selectedAccount = (bankAccounts ?? []).find((bank) => bank.id === invoiceCollectionsAccountID);
 
         return selectedAccount?.name ?? '';
     }, [bankAccounts, invoiceCollectionsAccountID]);
 
+    const selectedBillPaymentAccountName = useMemo(() => {
+        const selectedAccount = (bankAccounts ?? []).find((bank) => bank.id === reimbursementAccountID);
+
+        return selectedAccount?.name ?? '';
+    }, [bankAccounts, reimbursementAccountID]);
```

Comment:
NAB, both `selectedBillPaymentAccountName` and `selectedBankAccountName` have the same logic. Should we reuse it by creating a new function and passing `reimbursementAccountID` or `invoiceCollectionsAccountID` as a parameter? 

I think it can be like this:
```javascript
    const getSelectedBankAccount = useMemo((bankAccountId) => {
        const selectedAccount = (bankAccounts ?? []).find((bank) => bank.id === bankAccountId);

        return selectedAccount?.name ?? '';
    }, [bankAccounts]);
```